llm/llama-3/README.md

```diff
@@ -226,7 +226,7 @@ ENDPOINT=$(sky status --endpoint 8081 llama3)
 curl http://$ENDPOINT/v1/chat/completions \
   -H "Content-Type: application/json" \
   -d '{
-    "model": "meta-llama/Meta-Llama-3-8B-Instruct",
+    "model": "meta-llama/Meta-Llama-3-70B-Instruct",
```

Comment:
Is this intended?

Reply:
Yes, we are using 70B in the YAML above, it should be 70B here as well : )